src/frontend/screens/Settings/components/Gamescope.tsx

```diff
@@ -352,6 +353,29 @@ const Gamescope = () => {
           />
         </div>
       )}
+      {/* Additional Options */}
+      <TextInputField
+        label={t('options.gamescope.additionalOptions', 'Additional Options')}
+        htmlId="additionalOptions"
+        placeholder=""
+        value={gamescope.additionalOptions}
+        afterInput={
+          <FontAwesomeIcon
+            className="helpIcon"
+            icon={faCircleInfo}
+            title={t(
+              'help.gamescope.additionalOptions',
+              'Additional commandline flags to pass into gamescope.'
+            )}
+          />
+        }
+        onChange={(event: ChangeEvent<HTMLInputElement>) =>
```

Comment:
You might want to use `onBlur` instead of `onChange` here to only update the config when the user is *done* editing (clicks somewhere else)